lib/group/digest2/formatData.js

```diff
@@ -25,7 +25,7 @@ const presentPost = curry((slug, post) => {
   return pickBy(x => x, {
     id: post.id,
     title: post.get('name'),
-    details: RichText.qualifyLinks(post.get('description'), null, null, slug),
+    details: RichText.qualifyLinks(post.details && post.details(), null, null, slug),
```

Comment:
why check for post.details here (but not below in savedSearches)?

Reply:
Probably no good reason. About to replace `RichText` in the follow-on branch so will normalize this then. probably doesn't need to be checked at all but will confirm on that round.